YoutubeDownloader/Views/Dialogs/DownloadSingleSetupView.xaml

```diff
@@ -91,27 +85,23 @@
                     Width="150"
                     Margin="8,0,0,0"
                     VerticalAlignment="Center"
-                    materialDesign:ComboBoxAssist.ShowSelectedItem="True"
-                    IsReadOnly="True"
                     ItemsSource="{Binding AvailableDownloadOptions}"
                     SelectedItem="{Binding SelectedDownloadOption}">
+                    <!--materialDesign:ComboBoxAssist.ShowSelectedItem="True"-->
                     <ComboBox.ItemTemplate>
                         <DataTemplate>
                             <TextBlock>
                                 <!--  Video quality  -->
-                                <Run FontWeight="SemiBold">
-                                    <Run.Style>
-                                        <Style TargetType="{x:Type Run}">
-                                            <Style.Triggers>
-                                                <DataTrigger Binding="{Binding IsAudioOnly}" Value="False">
-                                                    <Setter Property="Text" Value="{Binding VideoQuality, Mode=OneWay}" />
-                                                </DataTrigger>
-                                                <DataTrigger Binding="{Binding IsAudioOnly}" Value="True">
-                                                    <Setter Property="Text" Value="Audio" />
-                                                </DataTrigger>
-                                            </Style.Triggers>
+                                <Run FontWeight="SemiBold"
+                                    Classes.audioOnly="{Binding IsAudioOnly}">
+                                    <Run.Styles>
+                                        <Style Selector="Run">
+                                            <Setter Property="Text" Value="{Binding VideoQuality, Mode=OneWay}" />
                                         </Style>
-                                    </Run.Style>
+                                        <Style Selector="Run.audioOnly">
```

Comment:
Is there no way to do something like `<Style Conditional={Binding IsAudioOnly}>`?

Reply:
Okay, fair enough